### PR TITLE
fix: wrong list type change (close #208)

### LINKED ISSUE #208: inconsistent list type replace
## Current Behavior

The attempt to change the type from task to ul,ol is failing.
Because it doesn't remove markdown task syntax.
WYSIWYG doesn't remove task class too.

### OL

* `*` -> `OL` -> `1.`
* `1.` -> `OL` -> `1.`
* `* [ ]` -> `OL` -> `1. [ ]`
* `1. [ ]` -> `OL` -> `1. [ ]`

### UL

* `*` -> `UL` -> `*`
* `1.` -> `UL` -> `*`
* `* [ ]` -> `UL` -> `* [ ]`
* `1. [ ]` -> `UL` -> `* [ ]`

### TASK

* `*` -> `TASK` -> `* [ ]`
* `1.` -> `TASK` -> `1. [ ]`
* `* [ ]` -> `TASK` -> `*`
* `1. [ ]` -> `TASK` -> `1.`

### OL

* `*` -> `OL` -> `1.`
* `1.` -> `OL` -> `1.`
* `* [ ]` -> `OL` -> `1.`
* `1. [ ]` -> `OL` -> `1.`

### UL

* `*` -> `UL` -> `*`
* `1.` -> `UL` -> `*`
* `* [ ]` -> `UL` -> `*`
* `1. [ ]` -> `UL` -> `*`

### TASK

* `*` -> `TASK` -> `* [ ]`
* `1.` -> `TASK` -> `1. [ ]`
* `* [ ]` -> `TASK` -> `*`
* `1. [ ]` -> `TASK` -> `1.`

### PR DESCRIPTION
**Pull request by kyuwoo-choi**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has a description of the breaking change

### Description
#208 
e2e는 배포 전 한번에 추가 작성할 예정.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/markdownCommands/ol.js [hunk 1]
```diff
@@ -44,7 +45,7 @@ const OL = CommandManager.command('markdown', /** @lends OL */{
 
       if (listManager.isListOrParagraph(line)) {
         if (isUlOrTask(line)) {
-          listManager.replaceLineText(doc, i, /[*-] /, `${ordinalNumber}. `);
+          listManager.replaceLineText(doc, i, /([-*]|[\d]+\.)( \[[ xX]])? /, `${ordinalNumber}. `);
```
Comment (jinwoo-kim-nhn) [inline src/js/markdownCommands/ol.js]: 요 정규식도 네이밍 해서 상수 처리하면, 어떤 정규식인지 더 쉽게 파악할 수 있을것 같습니다.

**Comment by jinwoo-kim-nhn** [comment]
수고하셨습니다. 완료입니다.

**Comment by seonim-ryu** [comment]
[6/15] 이견 없습니다~

File: src/js/markdownCommands/ol.js (hunk 1 above)
Comment (kyuwoo-choi) [reply src/js/markdownCommands/ol.js]: 옙!

**Pull request merged**